reports/012/reproreview_012.Rmd

```diff
@@ -0,0 +1,131 @@
+
+This report is part of the reproducibility review at the AGILE conference.
+For more information see [https://reproducible-agile.github.io/](https://reproducible-agile.github.io/).
+This document is published on OSF at [https://osf.io/wncsm/](https://osf.io/w7vph/).
+To cite the report use
+
+Koukouraki, E. (2022, June).  Reproducibility review of: Exploratory Analysis and Feature Selection for the Prediction of Nitrogen Dioxide.  [https://doi.org/10.17605/OSF.IO/WNCSM](https://doi.org/10.17605/OSF.IO/W7VPH)
+
+# Reviewed paper
+
+Iskandaryan, D., Di Sabatino, S., Ramos, F., and Trilles, S.: Exploratory Analysis and Feature Selection for the Prediction of Nitrogen Dioxide, AGILE GIScience Ser., 3, 6, [https://doi.org/10.5194/agile-giss-3-6-2022](https://doi.org/10.5194/agile-giss-3-6-2022)
+
+# Summary
+
+The paper evaluates the competence of selected features in the prediction of Nitrogen Dioxide with Machine Learning. For this reproduciblity review, the Figures and Tables of *Section 5 - Experiments and Results* were considered, while the Figures of *Section 3 - Exploratory Analysis* were not. The code of the corresponding analysis can be found in the following archived Github repository: [https://github.com/reproducible-agile/ExploratoryAnalysis_FeatureSelection](https://github.com/reproducible-agile/ExploratoryAnalysis_FeatureSelection). The data that is necessary to run the code can be found in the following Zenodo repository: [https://zenodo.org/record/6543074](https://zenodo.org/record/6543074). The reproduced results were in accordance with the ones reported in the paper, so the **reproduction** of the paper is considered **successful**.
+
+
+\clearpage
+
+# Reproducibility reviewer notes
+
+The code for the reproduction of the paper's analysis was provided as a Github repository in the *Data and Software Availability* Section of the paper and the necessary data to run the code was provided as a Zenodo repository in the README of the aforementioned Github repository. The link to the datasets is the following [https://zenodo.org/record/6543074](https://zenodo.org/record/6543074). After the reproducibility review, the Github repository was forked and archived under the official Reproducible AGILE Github organization. 
+
+The implementation of the analysis is handled mainly by 2 Jupyter notebooks, which were executed in the environment of Google Colaboratory ([https://colab.research.google.com/](https://colab.research.google.com/)). The data had to be either uploaded manually or externally mounted. The latter required the modification of the corresponding paths in the code. In order to reproduce all the reported results, we had to manipulate the code by commenting and uncommenting different chunks, something that is not ideal. Furthermore, previous experience with the Keras API was necessary in order to figure out how the manipulation should be properly handled. Instructions on how to use the notebooks were not provided with the README.
```

Comment:
Unless you are intrigued or there is a shortness of time, please ask the authors for better instructions instead of fixing everything yourself.